### DISCUSSION TITLE
Track Creator and Filter by Creator

### DISCUSSION THREAD

**Discussion by magliok-wwt** [microsoft/AzUrlShortener#489] [Q&A]
I would like track who creates URLs. 
Then have the ability to filter by creator.
Additionally, I'd like to restrict users to only see URLs they have created. 
 
Is this a feature, could this be done?

**Comment by fboucher** [comment]
I like the idea @magliok-wwt. Thank you for your suggestion.

**Reply by magliok-wwt** [reply]
Anything we could do to help, let us know!
